lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -269,8 +269,11 @@ def main():
         parameters_to_update["calibration_id"] = np.int32
 
     nodes_keys = get_dataset_keys(args.input_file)
-    if args.no_image:
-        nodes_keys.remove(dl1_images_lstcam_key)
+    # Remove the images key from the list of nodes to be copied: 
+    # it will be added later if so chosen by the user.
+    # Copying it here sometimes resulted in too bulky files, due
+    # to empty chunks.
+    nodes_keys.remove(dl1_images_lstcam_key)
```

Comment:
It probably doesn't play a large role for the file size, but I think the same applies to the parameters table. Maybe compare once with no images?

Reply:
I thought the same and had already tested this morning, it did not make a difference (I tried with images, but total size was identical to within bytes). I have no idea of why the difference. This is a one-file test, I cannot tell if it would be the same for all files.